rfc/9/index.md

```diff
@@ -0,0 +1,342 @@
+In other words, according to this specification, the OME-Zarr's root-level `zarr.json` MUST be located in the root of the ZIP archive and not in a subfolder within the ZIP archive.
+Potential problems (e.g. loss of data) resulting from "accidentally" extracting the ZIP archive in-place (e.g. using on-board tooling of some operating systems) can be alleviated by introducing a custom file extension (see below).
+
+To facilitate efficient storage and access of zipped OME-Zarr files, a set of essential ZIP/Zarr parameters are recommended in this RFC:
+
+- Use the ZIP64 format. This is the default in most modern tooling and enables the creation of single-file OME-Zarr larger than 4 GiB.
+- Disable ZIP-level compression. This avoids unnecessary compression of already compressed data (e.g. when using Zarr compression codecs) and makes it easier to directly conduct partial reads of the ZIP archive.
+- Use the Zarr sharding codec. This reduces the number of records in the central directory header.
+- Include all `zarr.json` files at the beginning of the file and at the beginning of the central directory header in a breadth-first order, starting with the root-level `zarr.json` as the first entry. This enables efficient metadata processing and discovery of the hierarchy structure.
+- Include an OME-Zarr-specific archive comment in the ZIP file header, indicating compliance with these recommendations. This further facilitates efficient data/metadata access and also allows for additional (optional/recommended) single-file metadata that may be specified in future OME-Zarr versions.
+
+This RFC explicitly prohibits embedding a zipped OME-Zarr file as subhierarchy of a parent OME-Zarr hierarchy.
+In particular this prohibits "recursive zipping", the embedding of a zipped OME-Zarr file within a parent zipped OME-Zarr file.
+This restriction may be revised in the future, especially in the light of the "collections" RFC.
+
+Furthermore, this RFC prohibits splitting up the ZIP archive into multiple files ("multi-volume archives"), in favor of directory-backed OME-Zarr and Zarr's sharding codec.
+
+Finally, this RFC also defines a new file extension to be used specifically with zipped OME-Zarr files.
+This should enable file type detection (in absence of a magic number), improve user experience (e.g. by enabling file type association), avoid "accidental" in-place extraction (e.g. using on-board tooling of some operating systems) and encourage the use of OME-Zarr-specific tooling for creating zipped OME-Zarr files (to follow the recommendations listed earlier).
+
+Note that this RFC does not - semantically or otherwise - restrict the data content of OME-Zarr hierarchies to be stored in zipped OME-Zarr files.
+
+## Specification
+
+Amend the specification with the following section:
+
+### Single-file OME-Zarr
+
+This section specifies how to store an OME-Zarr hierarchy within a single file.
```

Comment:
```suggestion
This section specifies how to store an OME-Zarr hierarchy within a single zip file.
```

Reply:
Zipped OME-Zarr files (header 4) are a specific realization (subheader) of Single-file OME-Zarr (header 3). See also https://github.com/ome/ngff/pull/316#discussion_r2496472880.